src/components/charts/BarChart.tsx

```diff
@@ -137,7 +137,7 @@ const BarChart = () => {
     setStartDate(value);
     const selectedDate = new Date(value);
     let defaultTime: string = "";
-    defaultTime = startTime ? formatTime(startTime) : formatTime(new Date());
+    defaultTime = formatTime(startTime ? startTime : new Date());
```

Comment:
Actually you can simplify it even further.
`defaultTime = formatTime(startTime || new Date());`